features/cellular/framework/targets/UBLOX/AT/UBLOX_AT.cpp

```diff
@@ -98,7 +104,7 @@ nsapi_error_t UBLOX_AT::init()
 
     nsapi_error_t err = NSAPI_ERROR_OK;
 
-#ifdef TARGET_UBLOX_C027
+#ifdef UBX_MDM_SARA_G350
```

Comment:
shouldn't this be `UBX_MDM_SARA_G3XX` one could have G340. It would than lead to `UBX_MDM_SARA_G350` not being used therefore `UBX_MDM_SARA_G3XX` would be enough to define

Reply:
Let me double check the differences and i will fix this.